charts/stargate/README.md

```
@@ -17,9 +17,9 @@ Formance Stargate gRPC Gateway
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
+| aws | object | `{"targetGroups":{"grpc":{"ipAddressType":"ipv4","serviceRef":{"name":"{{ include \"core.fullname\" $ }}","port":"{{ .Values.service.ports.grpc.port | default 3068 }}"},"targetGroupARN":"","targetType":"ip"}}}` | Aws Stargate target groups |
```

Markdownlint (0.37.0)

Expected: 4; Actual: 5; Too many cells, extra data will be missing
Table column count

(MD056, table-column-count)